app/assets/stylesheets/general/_titles.scss

```
@@ -47,6 +47,15 @@ h6 {
   font-size: $font-size-base;
 }
 
+h1 .header_anchor,
+h2 .header_anchor,
+h3 .header_anchor,
+h4 .header_anchor,
+h5 .header_anchor,
+h6 .header_anchor {
+  text-decoration: none;
+}
+
```

```suggestion
h1, h2, h3, h4, h5, h6 {
  .header_anchor {
    text-decoration: none;
  }
}
```
Consider utilizing the ability of SCSS to nest selection levels to avoid repetition while achieving the same thing.